packages/rspack-dev-server/package.json

```diff
@@ -18,8 +18,9 @@
     "./package.json": "./package.json"
   },
   "scripts": {
+    "prepare": "prebundle",
     "build": "tsc -b ./tsconfig.build.json",
-    "dev": "tsc -w -b ./tsconfig.build.json",
+    "dev": "tsc -b -w ./tsconfig.build.json",
```

Comment:
we need to add tsconfig#paths, alias `import 'webpack-dev-server'` to `import '../compiled/webpack-dev-server'`

Example: https://github.com/web-infra-dev/rspack/blob/main/packages/rspack/tsconfig.json#L7-L10

Reply:
fixed